frontend/src/components/Toggle/toggle.cy.tsx

```diff
@@ -0,0 +1,73 @@
+import { Form, Formik } from "formik"
+import 'tailwindcss/tailwind.css'
+import { Toggle } from './Toggle'
+
+const initialValues = {
+    title: ""
+}
+
+const submitForm = (values: typeof initialValues) => {
+    console.log(values);
+}
+
+const Wrapper: React.FC<{ children: React.ReactNode }> = ({ children }) => {
```

Comment:
@Zoe-Bot Das Problem habe ich auch: Warum muss ich da noch children definieren? Das ist doch in FC drin?

Reply:
Hab's seit React 18 sind `children` explicit props.

Die Entwickler erklären warum hier: https://solverfox.dev/writing/no-implicit-children/
Quick fix: https://stackoverflow.com/questions/59106742/typescript-error-property-children-does-not-exist-on-type-reactnode